addons/mail/static/tests/discuss/core/web/sidebar.test.js

```diff
@@ -75,8 +75,8 @@ test("unknown channel can be displayed and interacted with", async () => {
     await contains(".o-mail-DiscussSidebarChannel.o-active", { text: "Not So Secret" });
     await insertText(".o-mail-Composer-input", "Hello", { replace: true });
     await click(".o-mail-Composer-send:enabled");
-    await contains(".o-mail-Message", { text: "Hello" });
     await waitNotifications([env, "discuss.channel/new_message"]);
+    await contains(".o-mail-Message", { text: "Hello" });
```

Comment:
Why this should be changed? Order between the 2 await should be irrelevant here.